notebooks/Stormer-Verlet.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "932f3d29",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up required packages\n",
+    "using Pkg\n",
+    "Pkg.activate(@__DIR__)\n",
+    "Pkg.instantiate()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88a0891c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Load packages\n",
+    "using BSeries\n",
+    "using SymPy\n",
+    "using Latexify\n",
+    "using LinearAlgebra"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f7593a1e",
+   "metadata": {},
+   "source": [
+    "Here we study the Stormer-Verlet method applied to the Kepler Problem:"
```

Comment:
I would feel slightly better if you could search & replace Stormer → Störmer :wink: